### PR TITLE
Use @percy/cli from github for regression

### PR DESCRIPTION
**Pull request by samarsault**
Support trigger workflows via dispatch from CLI https://github.com/percy/cli/pull/1201 for regression

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -26,4 +26,18 @@ jobs:
         env:
           PERCY_POSTINSTALL_BROWSER: true
       - run: yarn build
+      - name: Set up @percy/cli from git
```
Comment (itsjwala) [inline .github/workflows/test.yml:29]: make `on:` similar to other repos. 

```yml
on: 
  push:
  workflow_dispatch:
    inputs:
      branch:
        required: false
        type: string
        default: master
```

Comment (itsjwala) [reply .github/workflows/test.yml:29]: is `L28` - `yarn build` required? I believe `yarn test:coverage` handles building as well?

**Review by itsjwala — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (samarsault) [reply .github/workflows/test.yml:34]: I think we need it

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -4,6 +4,11 @@ on:
     branches: [master]
   pull_request:
   workflow_dispatch:
+    inputs:
```
Comment (itsjwala) [inline .github/workflows/test.yml:7]:
```suggestion
  workflow_dispatch:
    inputs:
```

File: .github/workflows/test.yml (hunk 1 above)
Comment (itsjwala) [reply .github/workflows/test.yml:34]: can you try it once? since other repos doesn't have it

Comment (samarsault) [reply .github/workflows/test.yml:33]: Removed it

**Review by itsjwala — APPROVED** [review]

**Pull request merged**